components/emoji_picker/emoji_picker.jsx

```diff
@@ -367,16 +368,49 @@ export default class EmojiPicker extends React.PureComponent {
         } : this.state.categories[key];
     }
 
+    sortEmojis(emojis) {
+        const {recentEmojis: recentEmojisProps} = this.props;
+        const recentEmojis = [];
+        const emojisMinusRecent = [];
+
+        Object.values(emojis).forEach((emoji) => {
+            let emojiArray = emojisMinusRecent;
+            for (let i = 0; i < emoji.aliases.length; i++) {
+                if (recentEmojisProps.includes(emoji.aliases[i].toLowerCase())) {
+                    emojiArray = recentEmojis;
+                }
+            }
+
+            emojiArray.push(emoji);
+        });
+
+        const sortEmojisHelper = (a, b) => {
+            return sortEmojis(a, b, this.state.filter);
```

Comment:
I know this isn't the same as `this.sortEmojis` but still it makes one do a double-take because it's imported with the same name as this class method.